apps/hash-graph/lib/graph/src/snapshot/entity/channel.rs

```diff
@@ -64,15 +63,15 @@ impl Sink<EntitySnapshotRecord> for EntitySender {
         mut self: Pin<&mut Self>,
         entity: EntitySnapshotRecord,
     ) -> Result<(), Self::Error> {
-        self.account
-            .start_send_unpin(
+        self.owner
+            .start_send_unpin(OwnerId::Account(
                 entity
                     .metadata
-                    .record_id
-                    .entity_id
-                    .owned_by_id
+                    .custom
+                    .provenance
+                    .record_created_by_id
```

Comment:
I'm not familiar with the context of this function – here we've switched from taking an `owned_by_id` (a user or an org) to the record creator (a user, not necessarily the owner) – can you confirm that's correct for its usage?

Reply:
This function is called when restoring a snapshot. The snapshot currently does not export accounts but instead creates account IDs in the Graph based on required accounts in the snapshots.

The previous behavior was to use the `owned_by_id` to create accounts that were referred to by entities. This is no longer possible as `owned_by_id` could be both, an account id or an account group id.  Ideally, this snippet would be removed entirely, but as long as we don't make accounts and account groups a part of the snapshot (I'm currently doing exactly this) there is no other easy way to seed the Graph with accounts. For the time being, I use the `record_created_by_id` to seed accounts and tested that with all available test sets I have.

I also left this snippet in to maintain compatibility between snapshots in the short term to be able to import older snapshots. With the future PR, which will export accounts and account groups as well, we can remove this.

Reply:
Understood, thank you.